src/views/Home/Home.test.jsx

```diff
@@ -10,6 +12,20 @@ const user = {
   color: 'crimson',
 }
 
-test('Should render the user profile', () => {
+test('Should render the user profile', async () => {
+  render(<Home user={user} />)
 
+  const name = await screen.findByText(/vonta/i)
```

Comment:
You only need to await one of these. for the rest you can use the `getBy` syntax